libcodechecker/util.py

```diff
@@ -128,10 +128,14 @@ def get_binary_in_path(basename_list, versioning_pattern, env):
         return files[0]
 
 
-def call_command(command, env=None):
+def call_command(command, env=None, cwd=None):
     """ Call an external command and return with (output, return_code)."""
 
     try:
+        old_cwd = os.getcwd()
+        if cwd:
+            os.chdir(cwd)
+
         LOG.debug('Run ' + ' '.join(command))
         out = subprocess.check_output(command,
```

Comment:
check_output has a `cwd` parameter, saving and restoring the old directory should not be required.